Resources/Locale/en-US/loadouts/jobs/heads/headOfPersonnel.ftl

```diff
@@ -0,0 +1,4 @@
+loadout-description-LoadoutCommandHOPNeckMantle = To show who has the authority around here.
+loadout-description-LoadoutcommandHOPNeckCloak = To really show who has the authority around here.
+loadout-description-LoadoutcommandHOPBackIan = A backpack that looks like Ian, how cute!
+loadout-description-LoadoutCommandHOPHatCap = The HOP's cap, pretty nice.
```

Comment:
As an example of the above the Command department isn't consistent here and is just going to be prone to bugs so this should not be done.